cluster/proto/cluster/request.go

```diff
@@ -27,7 +27,7 @@ type UpdateClassRequest struct {
 }
 
 type AddPropertyRequest struct {
```

Comment:
non-blocking(thoughts): Would it make sense to rename this to `AddPropertiesRequest` ?